src/app/components/case/CaseDetails/CaseDetails.tsx

```diff
@@ -27,13 +27,13 @@ const StyledDiv = styled.div`
 
 const getDataFirstCol = (caseItem?: Components.Schemas.Case) => {
   if (caseItem === undefined) return
-  const { id, theme, reason, start_date, sensitive, previous_case } = caseItem
+  const { id, theme, start_date, sensitive, previous_case, is_enforcement_request } = caseItem
   const data: any = {
     "Zaak ID": (
       <Wrap>
         <CaseIdDisplay id={ id } />
         <SensitiveCaseIcon sensitive={ sensitive }/>
-        <EnforcementIcon caseReason={ reason } />
+        {is_enforcement_request && <EnforcementIcon />}
```

Comment:
Please do the check/validation inside the component. `<EnforcementIcon isEnforcementRequest={ is_enforcement_request } />`